admin.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+
+
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Bens Admin Page</title>
+    <link rel='stylesheet' type='text/css' href='css/normalise.css'>
+    <link rel='stylesheet' type='text/css' href='css/admin.css'>
+</head>
+
+<body>
+    <header>
+        <h1>Admin Page</h1>
+        <h2>Welcome Ben</h2>
+        <h3>About Me</h3>
+    </header>
+    <div class="forms">
+        <div class="container">
+            <h4>Add information</h4><form method="post">
+                <textarea rows="4" cols="100" name="add"></textarea>
+                <input type="submit" value="Add">
+            </form>
+        </div>
+        <div class="container">
+            <h4>Edit information</h4>
+            <p>Select Paragraph to Edit</p>
+                <select>
```

Comment:
I assume this drop down is going to be dynamic? And populated from the database? If so I would suggest to add all this html to a dummy php variable, then include that on the page, because you're going to have to do that anyway at some point, just make it dynamic? And you'll not be keeping this html?